test/tests/AmazonSESTest.php

```diff
@@ -4,18 +4,18 @@
 
 use Omnimail\Exception\Exception;
 use Omnimail\Email;
-use Omnimail\AmazonSES;
+use Omnimail\Omnimail;
 
 class AmazonSESTest extends BaseTestClass
 {
     public function testErrorMessageIsThrownWithIncorrectDetails()
     {
         $this->expectException(Exception::class);
 
-        $accessKey = 'ACCESSKEY';
-        $secretKey = 'SECRETKEY';
-
-        $sender = new AmazonSES($accessKey, $secretKey);
+        $sender = Omnimail::create(Omnimail::AMAZON_SES, [
+            'accessKey' => 'ACCESSKEY',
+            'secretKey' => 'SECRETKEY',
+        ]);
```

Comment:
The initialize method seemed redundant or unnecessary, so I removed it and added parameters to the create method instead.